src/app/features/home/post-capture-tab/post-capture-tab.component.ts

```diff
@@ -51,28 +51,37 @@ export class PostCaptureTabComponent implements OnInit {
     startWith(undefined)
   );
 
+  private readonly refreshPostCaptures$ = combineLatest([
+    this.focus$,
+    this.onDidNavigate$,
+  ]).pipe(
+    filter(([focus]) => focus),
+    switchMapTo(defer(() => this.fetchPostCaptures$())),
+    catchError(err => {
+      console.error(err);
+      return VOID$;
+    })
+  );
+
+  private readonly prefetchSize = 9;
+
+  private readonly prefetchPostCaptures$ = this.fetchPostCaptures$(
```

Comment:
See https://github.com/numbersprotocol/capture-lite/blob/66751fb9b7301445801c7b466b5745b4840f245b/src/app/shared/services/dia-backend/asset/dia-backend-asset-repository.service.ts#L133
It's a trap (hint: it's always falsy)

The line makes `is_original_owner` query parameter not added to the requests. And the first 9 assets ordered by `source_transaction` might not all be PostCaptures, so the prefetch won't work as expected because of the lacking of the `is_original_owner` param.